conf.d/metrics.conf

```diff
@@ -1,7 +1,7 @@
 [program:collector]
 command=twistd --nodaemon
     --pidfile=%(program_name)s.pid vumi_worker
-    --worker-class vumi.blinkenlights.GraphiteMetricsCollector
+    --worker-class vumi.blinkenlights.UDPMetricsCollector
```

Comment:
I think for this, we'll need some additional config options, like
`--set-option=metrics_host:localhost`
`--set-option=metrics_port:9109`
`--set-option=format_string:%(metric_name)s %(value)s %(timestamp)s\n`
`--set-option=timestamp_format:%s`

Reply:
okay, thanks